### PR TITLE
Fix minor nits on the security tasks page Plugin CA Certificate

### PR DESCRIPTION
**Pull request by majeshps**
Partially fixes: #12695

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 1]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy for the foo namespace:
```
Comment (ericvn) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:198]:
```suggestion
*   Remove the authentication policy from the foo namespace:
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 2]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
...
+    {{< /text >}}
+
+*   Remove the sample and httpbin application:
+
+    {{< text bash >}}
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+    {{< /text >}}
+
+*   Remove the Istio Components from the cluster:
```
Comment (ericvn) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:211]:
```suggestion
*   Remove Istio from the cluster:
```

I'm inclined to suggest that verbs in each of the bullets be updated to follow the verb in the command, example for this change Remove to Uninstall.

Comment (johnma14) [reply content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:211]: So the recommendation is to say:
- Uninstall sample applications `httpbin and `sleep`
- Uninstall Istio from the cluster

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md (hunk 1 above)
Comment (frankbu) [reply content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:198]: `for` is equally correct in this case because it is "the policy for the foo namespace" that's being delete (from the foo namespace).

**Review by Arhell — COMMENTED** [review]
Please sign CLA

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 3]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
...
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy for the foo namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample and httpbin application:
+
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:205]: It should be something like : Remove the sample applications `sleep` and httpbin`:

**Comment by majeshps** [comment]
@ericvn @johnma14 The proposed changes are completed.
@Arhell The CLA signed as well

**Comment by kfaseela** [comment]
Please run `make gen` locally to get the auto-gen files. Also run `make lint` locally to fix the lint failures

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 4]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
...
     {{< /text >}}
 
-*   To remove the Istio components: follow the [uninstall instructions](/docs/setup/getting-started/#uninstall) to remove.
+*   Remove the authentication policy from the foo namespace:
+
+    {{< text bash >}}
+    $ kubectl delete peerauthentication -n foo default
+    {{< /text >}}
+
+*   Remove the sample applications sleep and httpbin:
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:204]: Can we rephrase this to: Delete sample applications `sleep` and `httpbin` from the `foo` namespace: (Put the names in backpacks ``)

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 5]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
     $ kubectl delete ns foo istio-system
     {{< /text >}}
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:196]: Need to remove this command from the beginning. We moved this to the end after doing all the other cleanup commands.

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 6]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
...
+
+*   Remove the sample applications sleep and httpbin:
+
+    {{< text bash >}}
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+    {{< /text >}}
+
+*   Remove the Istio from the cluster:
+
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:212]: Rephrase this to: Uninstall Istio from the cluster:

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 7]
```diff
@@ -195,4 +195,27 @@ openssl command is expected.
...
+    {{< text bash >}}
+    $ istioctl uninstall --purge
+    {{< /text >}}
+
+*   Remove the namespace foo and istio-system from the cluster:
+
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:218]: put the names `foo`, `istio-system` in back ticks ``.

**Review by johnma14 — COMMENTED** [review]
Some more changes requested.

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 8]
```diff
@@ -188,11 +188,33 @@ openssl command is expected.
...
+    {{< text bash >}}
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+    {{< /text >}}
+
+*   Uninstall Istio from the cluster:
+
+    {{< text bash >}}
+    $ istioctl uninstall --purge
```
Comment (johnma14) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:213]: Change to `istioctl uninstall --purge -y`

**Review by johnma14 — COMMENTED** [review]
some more changes

**Comment by majeshps** [comment]
@johnma14 corrected and updated the above changes.

**Comment by johnma14** [comment]
Changes look good. Thanks @majeshps.

**Comment by johnma14** [comment]
@ericvn @kfaseela @frankbu could you take a look at this? I think this is ready. Thanks.

**Comment by johnma14** [comment]
should we keep this or close it?

**Review by ericvn — APPROVED** [review]
I'm fine with this change as we are explicitly removing things from the namespaces instead of just deleting them (and all things in them). Allows the user to decide to only remove some things leaving the namespaces around if desired.
